binlog-stream/app/io/laserdisc/mysql/binlog/stream/MysqlBinlogStream.scala

```diff
@@ -48,7 +48,7 @@ object MysqlBinlogStream {
       client: BinaryLogClient
   ): Stream[F, Event] =
     for {
-      d <- Stream.resource(Dispatcher[F])
+      d <- Stream.resource(Dispatcher.parallel[F])
```

Comment:
The `apply` call was deprecated in cats effect 3.4 (`parallel` is equivalent to the old `apply`)